pkg/util/util.go

```diff
@@ -221,11 +221,11 @@ func Md5sum(filePath string) (string, error) {
 
 // GetUsableSpace calculates usable space to use taking file system overhead into account
 func GetUsableSpace(filesystemOverhead float64, availableSpace int64) int64 {
-	// +1 always rounds up.
-	spaceWithOverhead := int64(math.Ceil((1 - filesystemOverhead) * float64(availableSpace)))
+	// Reverse the overhead calculation
+	spaceWithoutOverhead := int64(math.Ceil(float64(availableSpace) / (1 + filesystemOverhead)))
```

Comment:
So I suppose this was less about undoing the inflating but rather the original filesystem overhead mechanism which just took away a certain percentage of the user request
(Back then there was no .storage API, only .pvc, so you get a PVC with same size as your request)